API_CHANGES.md

```diff
@@ -1,3 +1,24 @@
+# 10.2
+
+## New features
+### Analysis and tracking
+
+* Add `analyzeFilesAndTrack` method to `org.sonarsource.sonarlint.core.rpc.protocol.backend.analysis.AnalysisRpcService` to allow client to submit files for analysis and expect to get a notification with tracked issues.
+  * should be called by the client instead of the deprecated `analyzeFiles` method.
+  * accepts `AnalyzeFilesAndTrackParams` instead of deprecated `AnalyzeFilesParams`. The difference is that the new `AnalyzeFilesAndTrackParams` expects to get one more flag - `shouldFetchServerIssues`.
+
+* Add `raiseIssues` method to `org.sonarsource.sonarlint.core.rpc.protocol.SonarLintRpcClient` to report tracked issues.
+  * will be called by core after tracking is done.
+  * returns `org.sonarsource.sonarlint.core.rpc.protocol.client.issue.RaiseIssuesParams` that contains new data class `TrackedIssueDto`.
+  * `TrackedIssueDto` contains data from matching/tracking with previously analyzed issues. For connected mode - also has data from matched server issues and anticipated issues.
+
+## Deprecation
+* `analyzeFiles` method of `org.sonarsource.sonarlint.core.rpc.protocol.backend.analysis.AnalysisRpcService` should be called. `analyzeFilesAndTrack` should be called instead.
+* `trackWithServerIssues` method of `org.sonarsource.sonarlint.core.rpc.protocol.backend.tracking.IssueTrackingRpcService` should be called if the client migrated to a new analysis API. Tracking will be performed by the core internally.
+* Implementation of the `didRaiseIssue` method of `org.sonarsource.sonarlint.core.rpc.protocol.SonarLintRpcClient` is no longer required. Instead new method `raiseIssues` should be implemented.
+  * Note that the new method `raiseIssues` reports a collection of issues and works like issue publishing. Every call contains the full list of known issues. So it should be published as a whole instead of previously published issues.
+
```

Comment:
I would move this note in the first section about "new features". I think it's a bit hidden here in the "deprecations" section